src/modules/site/constants.py

```diff
@@ -7,6 +7,7 @@ class PRICES:
   STRAIN = 15
   SHIPPING = 65
 
+JWT_LIMITED_TOKEN_EXPIRES = 15 * 60

```

Comment:
Consider prefixing with `PASSWORD_RESET_` and moving this to the `global.env` under `env/main/global.env`. Add it to the `site` section of `global.env`.